components/services/bat_ledger/public/interfaces/bat_ledger.mojom

```diff
@@ -179,4 +180,5 @@ interface BatLedgerClient {
   ConfirmationsTransactionHistoryDidChange();
 
   GetExcludedPublishersNumberDB() => (uint32 number);
+  DeleteClientStateFiles() => (bool success);
```

Comment:
The LedgerClient shouldn't have to know what needs to happen to delete the client state. That is likely to lead to inconsistent implementations. We already have `ResetState` methods for the different state files and I am adding delete methods for the DB